test/test_stream.py

```diff
@@ -86,12 +86,12 @@ def write_objs2(fpath, *objs_list):
         fpath (string): path of the file to be written.
         objs_list (*protobuf.message.Message): list of objects to be written.
     """
-    ostream = stream.open(fpath, 'wb', buffer_size=(len(objs_list)//2))
+    ostream = stream.open(fpath, 'wb', buffer_size=(len(objs_list)//2), gzip=kwds.get('gzip', True))
```

Comment:
This line seems to be too long. Please limit all lines to a maximum of 79 characters.